mlrun/model_monitoring/helpers.py

```diff
@@ -294,6 +295,112 @@ def _get_profile(
 )
 
 
+def _get_v3io_output_stream(
+    *,
+    v3io_profile: mlrun.datastore.datastore_profile.DatastoreProfileV3io,
+    project: str,
+    function_name: str,
+    v3io_access_key: Optional[str],
+    mock: bool = False,
+) -> mlrun.platforms.iguazio.OutputStream:
+    stream_uri = mlrun.mlconf.get_model_monitoring_file_target_path(
+        project=project,
+        kind=mm_constants.FileTargetKind.STREAM,
+        target="online",
+        function_name=function_name,
+    )
+    endpoint, stream_path = mlrun.platforms.iguazio.parse_path(stream_uri)
+    return mlrun.platforms.iguazio.OutputStream(
+        stream_path,
+        endpoint=endpoint,
+        access_key=v3io_profile.v3io_access_key or v3io_access_key,
```

Comment:
I think you will have a problem because all the streams beside the controller are on the project container while the controller stream is on the user container(pipelines)

Reply:
I reversed the variables' order as we discussed